core/server/data/migration/006/index.js

```diff
@@ -0,0 +1 @@
+module.exports = [];
```

Comment:
Not sure i understand. This part of 006 is for now empty. The new script is in `fixtures`.